virttest/qemu_monitor.py

```diff
@@ -22,13 +23,23 @@
     logging.warning("Could not import json module. "
                     "QMP monitor functionality disabled.")
 
-from . import passfd_setup
 from . import utils_misc
 from . import cartesian_config
 from . import data_dir
 
 from virttest.qemu_capabilities import Flags
 
+if sys.version_info[:2] >= (3, 0):
```

Comment:
well there is no need in comparing the second number, pure `sys.version_info[0]` would do, but it's not a big issue so let's just leave it as is unless I find something else...